mysql-test/suite/tokudb/include/setup_fast_update_upsert.inc

```diff
@@ -0,0 +1,2 @@
+set session sql_mode=(select replace(@@sql_mode,'STRICT_TRANS_TABLES',''));
```

Comment:
Previous question still standing: how is this setting up fast update/upserts? Why STRICT_TRANS/ALL_TABLES have to be removed?

Reply:
Because it's supposed that for strict mode some additional checks must be provided, what can lead to necessity to read some data from storage, while the general optimization for fast update and upsert is that there are no read operations during insert/update execution.

Reply:
This needs to be documented in a comment